docs/en/advanced/Security.md

```diff
@@ -11,13 +11,15 @@ priority: 0
 
 This document describes the following security related features in Alluxio.
 
-1. [Authentication](#authentication): If `alluxio.security.authentication.type=SIMPLE` (by default),
+1. [Authentication](#authentication): 
+TODO: I can't even begin to parse this...
```

Comment:
i may know how to parse this now, after reading the rest of the doc...

Reply:
rephrased a bit